docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG swift_version=5.0
+ARG swift_version=nightly-5.5
 ARG ubuntu_version=bionic
-ARG base_image=swift:$swift_version-$ubuntu_version
+ARG base_image=swiftlang/swift:$swift_version-$ubuntu_version
```

Comment:
this will need to be reversed so we can support both release versions and nightlies